kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -270,7 +270,7 @@
       facilityOptions(facility) {
         return [
           {
-            label: this.coreString('registerAction'),
+            label: 'Manage sync Schedule',
```

Comment:
This string needs to be added as a translatable string!

Also, you have just overwritten the register action option here, you should be adding an additional option alongside `register`.